README.md

```diff
@@ -1,12 +1,18 @@
 <div align="center">
-  <img width="100px" src="amsterdam-schema.svg" />
+  <img alt="Amsterdam logo" width="100px" src="amsterdam-schema.svg" />
 </div>
 
 # Amsterdam Schema
 
-Amsterdam Schema aims to describe and validate [open data published by the City of Amsterdam](https://api.data.amsterdam.nl/api/), in order to make the storing and publishing of different datasets more structured, simpler and better documented.
+Amsterdam Schema aims to describe and validate
+[open data published by the City of Amsterdam](https://api.data.amsterdam.nl/api/),
```

Comment:
Je hebt deze laten staan omdat we op termijn de catalogus op deze url ook toegankelijk haan maken https://api.data.amsterdam.nl/api/?

Reply:
Actually this appeared in the diff due to some reformatting work (https://rhodesmill.org/brandon/2012/one-sentence-per-line/) in prep op adding new documentation. I have no idea if this part is still relevant or what in entails for the future.

Reply:
Leaving it in for now. Not really part of the PR.